File: C/recursion.c/factorial.c
#include<stdio.h>
int fact(int n)
{
    if(n>1)
    {
        return n*fact(n-1);
    }
}
void main()
{
    int num;
    printf("Enter any number\n");
    scanf("%d",&num);
    printf("Factorial of %d is %d",num,fact(num));
}

File: Java/JAVA 3.0/RECURSION/factorial.java
package RECURSION;

public class factorial {
    static int fact(int num)
    {
        if(num == 1)
        {
            return 1;
        }else{
            return num * fact(num-1);
        }
    }

    public static void main(String[] args) {
        int fact = fact(5);
        System.out.println(fact);
    }
}
